marlowe-dashboard-client/tailwind.config.js

```diff
@@ -19,12 +19,17 @@ module.exports = {
       lightblue: "#ccddea",
       yellow: "#fed130",
       white: "#fff",
-      gray: "#ebebeb"
+      gray: "#ebebeb",
+      transgray: "rgba(10,10,10,0.1)"
     },
     spacing: {
       0: "0px",
       0.5: "0.5rem",
-      1: "1rem"
+      0.75: "0.75rem",
+      1: "1rem",
```

Comment:
To keep it consistent with tailwind... I'd suggest that if we need 2rem spacing, we use the size 8 and if we need 3rem spacing we use the size 12

```
  spacing: {
        px: '1px',
        0: '0px',
        0.5: '0.125rem',
        1: '0.25rem',
        1.5: '0.375rem',
        2: '0.5rem',
        2.5: '0.625rem',
        3: '0.75rem',
        3.5: '0.875rem',
        4: '1rem',
        5: '1.25rem',
        6: '1.5rem',
        7: '1.75rem',
        8: '2rem',
        9: '2.25rem',
        10: '2.5rem',
        11: '2.75rem',
        12: '3rem',
        14: '3.5rem',
        16: '4rem',
        20: '5rem',
        24: '6rem',
        28: '7rem',
        32: '8rem',
        36: '9rem',
        40: '10rem',
        44: '11rem',
        48: '12rem',
        52: '13rem',
        56: '14rem',
        60: '15rem',
        64: '16rem',
        72: '18rem',
        80: '20rem',
        96: '24rem',
      },
```

Reply:
and card could be `80`... it's weird to have a spacing card, as even if we are using it as width, you are also generating padding and margin `card`... and that doesn't make too much sense.

Also, if you are specifying width's that large, maybe it make sense to use percentage measures for cards width... if you want 4 cards in medium you could use `md:w-1/4`

Reply:
Hmmm, not sure about the spacing labels, but we can revisit this when we have the visual designs to implement. I'd much rather have consecutive labels. I don't want to have to keep checking which labels are defined and remember that there are gaps.

There will be some full screen cards (minus a bit of margin) in due course, but I haven't got that far yet. Will worry about it later.

Reply:
_backs away_